tests/unix/test_lwt_unix.cppo.ml

```diff
@@ -677,11 +677,28 @@ let bind_tests = [
         (fun () -> Lwt_unix.close socket));
 ]
 
+let dir_tests = [
```

Comment:
You should probably work with the jbuilder build system is `master`, because we won't be able to merge anything that uses `setup.ml`. The only reason I'm aware of at this point to use the older build system is coverage, but I hope to have that solved in a couple days.

One way to address this is:

1. We have this helper: https://github.com/ocsigen/lwt/blob/ba4deb79166235cfe32d5515dd99adc56793274d/tests/test.mli#L52. I think it's okay not to clean up the temporary directory, though I'm not fully sure how this interacts with the new jbuilder build system.
2. Ignore this :) With the test written using the helper, it shouldn't be a problem.